### PR TITLE
apack: fix namespace problem

### PR DESCRIPTION
**Pull request by einareiriksson**
Add namespace handling to copy_manifest method. Extract package from manifest instance and use namespace if present.

### CODE REVIEW

**Comment by einareiriksson** [comment]
@mbtools this fixes https://github.com/abapGit/abapGit/issues/6837

File: src/apack/zcl_abapgit_apack_reader.clas.abap [hunk 1]
```diff
@@ -98,6 +102,17 @@ CLASS zcl_abapgit_apack_reader IMPLEMENTATION.
     lv_descriptor_cust = zif_abapgit_apack_definitions=>c_apack_interface_cust && '~DESCRIPTOR'.
     lv_descriptor_sap  = zif_abapgit_apack_definitions=>c_apack_interface_sap && '~DESCRIPTOR'.
 
+    lv_class_name = cl_abap_classdescr=>get_class_name( io_manifest_provider ).
+    SPLIT lv_class_name AT '\CLASS=' INTO lv_empty lv_class_name.
+
+    IF lv_class_name(1) = '/'.
+      CREATE OBJECT lo_nspc.
```
Comment (mbtools) [inline src/apack/zcl_abapgit_apack_reader.clas.abap:109]: let's replace this with `zcl_abapgit_factory=>get_sap_namespace( )->split_by_name( ... )`

**Review by mbtools — CHANGES_REQUESTED** [review]
small change

File: src/apack/zcl_abapgit_apack_reader.clas.abap (hunk 1 above)
Comment (einareiriksson) [reply src/apack/zcl_abapgit_apack_reader.clas.abap]: Sounds good.

**Comment by mbtools** [comment]
tested with https://github.com/abapGit-tests/NSPC_APACK 👍

**Comment by mbtools** [comment]
Welcome to abapGit as a contributor 🎉

**Pull request merged**